code-challenges.test.js

```diff
@@ -1,59 +1,223 @@
 
-// a) Create a test with expect statements for each of the variables provided.
+
+
+// // _________________________THIS WAS WORK PRIOR TO WORKING WITH INSTRUCTOR____________________________________________________
+// // //     if(object % 3 === 0 ){
+// // //         return `${object} is divisible by three`
+// // //     } else if (object % 3 != 0){
+// // //         return `${object} is not divisible by three`
+// // //     }
+  
+
+
+// //   const object = [15, 0, -7] => {
+// //     if (object % 3 === 0){
+// //         return `${object}` is divisible by three`
+// //     } else if(object )
+// //         return `${object}` is not divisible by three`
+// //     return `${object}` is divisible by three`
+
+
+
+// // describe("object", () => {
+// //     it("is divisible by three", () => {
+// //       let num = [15 , 0, -7]
+// //       let expectedOutput = ["15 is divisible by three", "0 is divisible by three", "-7 is not divisible by three"]
+// //       expect(object(num, expectedOutput)).toEqual(expectedOutput)
+// //     })
+// //   })
+
+
+// // //     // function isDivisibleBy3(num){ if ((num % 3) == 0){ return true; } else { return false; } };
+
+// // // // When working on this all my work came back undefined, which is typically what we want but when doing my function is not working and that object wasnt defined . I have to remember to read the question for Context. I went through the syllabus not thinking when you meant object you meant object. 
+// // // I find myself thinking really hard, into the jest doing its own math, when we give it what to say and think. My mind is overflowing with console.log. 
+
+// // _________________________________________________________________________________________________________________________________________________________
+
+
+// // // --------------------2) 
 
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+// // // // // // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // // // // // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+
+// // Create a function that takes in an array of words and returns an array with all the words capitalized.
+
+
+// // // a) Create a test with expect statements for each of the variables provided.
+
+describe("randomNouns", () => {
+    it("an array with all the words capitalized", () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+      expect(randomNouns(randomNouns1)).toUpperCase("STREETLAMP", "POTATO", "TEETH", "CONCLUSION", "NEPHEW")
```

Comment:
Tests should not have any logic or built-in methods used in them.  They are simply a tool to invoke the function and compare the output from the function to a predetermined output.